dotcom-rendering/src/web/components/Dropdown.tsx

```diff
@@ -177,58 +182,50 @@ const buttonExpanded = css`
 	}
 `;
 
-const badgeDiameter = 20;
 const notificationColor = palette.error[400];
 
-const notificationBadgeStyles = css`
+const notificationBadgeStyles = (diameter: number) => css`
 	background-color: ${notificationColor};
 	color: ${palette.neutral[100]};
-	position: absolute;
-	top: 0;
-	left: 0;
-	min-width: ${badgeDiameter}px;
-	height: ${badgeDiameter}px;
-	border-radius: ${badgeDiameter / 2}px;
+	margin-top: 12px;
 	text-align: center;
 	display: flex;
 	justify-content: center;
 	align-items: center;
-	padding: 0 5px;
-	margin-left: -10px;
-	margin-top: -3px;
-`;
+	${textSans.xsmall()};
+	line-height: 1;
 
-const notificationStyles = css`
-	${textSans.xxsmall()};
-	color: ${notificationColor};
+	min-width: ${diameter}px;
+	height: ${diameter}px;
+	border-radius: ${diameter / 2}px;
```

Comment:
I'm assuming that this can be made purely square, now. You also do not have to halve the `border-radius` as CSS will happily handle larger values.

Reply:
we still need it to be round!

Reply:
Haha I wrote square but I mean “round” as opposed to “pill”. You no longer need the width to be `min-width`, it can be explicitly `width`.